src/appleseed.studio/python/module.cpp

```diff
@@ -68,7 +68,7 @@ Project* new_project()
 
 Project* open_project(const char* project_path)
```

Comment:
Sorry, it's MainWindow::do_load_project the method that returns a bool.
Only in the new open_project sync method, can we return this bool to python?